content/08.outlook.md

```diff
@@ -1 +1,45 @@
 ### Outlook
+
+Throughout this perspective we have highlighted various challenges in applying ML methods to rare disease data as well as examples of approaches that address these challenges.
+Scarcity of samples while significant is not the only roadblock towards application of ML in rare disease data.
+The high dimensionality of the data collected using modern techniques necessitates the management of the curse of dimensionality in creative ways by learning low dimensional representations from the data.
+It further requires leveraging prior knowledge and transfer learning instead of using traditional machine learning methods for appropriate interpretation of data. 
+Additionally, anyone applying machine learning methods on rare disease data is advised to delve into fundamental techniques that increase confidence and penalize complexity of the resultant models to enhance generalizabilty of ML in rare disease.
+
+All of the approaches highlighted in this perspective come with certain challenges or inadequacies that breed mistrust in using these powerful techniques in rare disease.
+We believe that the same challenges that are currently considered major pitfalls in applying ML to rare disease can be great opportunities for data generation as well as method development in moving the field forward.
+During our journey through the various challenges, we identified two major areas where mindful strategies can immeasurably enhance the power of machine learning in rare disease and move the field forward.
+
+_Emphasis on not just "more n" but "more meaningful n"._
+
+Mindful addition of data is key for powering the next generation of analysis in rare disease data.
+While there are many techniques to collate rare data from different sources, incorrect data generation may hurt the end goal even if it adds to the size of the dataset.
+In our experience collaboration with domain experts have proved to be critical in gaining insight into potential sources of variation in the datasets.
+As an example, an neurofibromatosis type 1 (NF1) dataset was found to contain samples collected using vastly different surgical techniques (laser ablation and excision vs standard excision). [@doi:10.3390/genes11020226] 
+While the integrative analysis in the study using transfer learning techniques was able to minimize technique related signals [@doi:10.1016/j.cels.2019.04.003], a more traditional analysis may have resulted in surfacing of substantial biological differences that are a consequence of process (e.g. activation of heat shock protein related pathways), not disease related biology. 
```

Comment:
This is based on some of the discussion during writing of the earlier manuscript that since the latent variable generation finds patterns that are consistent among datasets that have different technical differences in them,  the result is enrichment of LVs that are biologically relevant (since the technical differences would presumably be orthogonal in different datasets). 
But maybe I am misremembering... Tagging @cgreene and @jaclyn-taroni for more insight

Reply:
Also.. happy to have you tackle this section after merging :)

Reply:
Alrighty, filed as #143!